### ISSUE TITLE
typo in compute_gradient_in_acoustic.f90

### ISSUE THREAD

**Issue by llanger** [geodynamics/specfem3d#1396]
Line 101 says 'do l = 1,NGLLZ' where it should say 'do l = 1,NGLLY'. The number of GLL points is generally the same regardless of dimension so most likely this doesn't have a real impact, but it may be worth fixing for greater readability.

**Comment by danielpeter** [comment]
right, good catch! for that generic case, better to loop over NGLLY to keep things general. will be corrected in the next PR.
